.travis.yml

```diff
@@ -24,6 +24,7 @@ jdk:
   - oraclejdk8
 
 services:
+  - docker
```

Comment:
so I think the real real test is removing the manual installation of elasticsearch further down this file and seeing if the tests still run